client/src/components/App.jsx

```diff
@@ -45,6 +50,12 @@ export default class App extends React.Component {
       moreReviewBtn: reviews.length <= 2 ? false : true
       }))
     });
+
+    this.fetchMeta()
```

Comment:
is there a reason for defining the fetchMeta method? 
it seems unnecessary for one line of code, and if we did that for each request it would make the file a bit longer. maybe it's alright to just put the axios request here directly instead of invoking a method?

Reply:
Good point! I will update my code to reflect your suggestion.